taxonomies/food/ingredients.txt

```diff
@@ -2113,6 +2113,7 @@ ko:발효유
 lv:Skābpiena produkti
 ru:Кисломолочные продукты
 uk:Кисломолочний продукт
+nova:en:4
```

Comment:
I don't think "fermented milk products" should be NOVA 4. e.g. plain yogurt is NOVA 1.

Reply:
so should be set by children

Reply:
removed